external-crates/move/crates/move-compiler/src/hlir/translate.rs

```diff
@@ -277,8 +287,16 @@ impl<'env> Context<'env> {
                 &mdef.enums,
             );
         }
+        let debug = HlirDebugFlags {
+            match_translation: false,
+            match_variant_translation: false,
+            match_specialization: false,
+            match_counterexample: false,
+            match_work_queue: false,
```

Comment:
should we come up with a way of setting these without recompilation?

Reply:
These could eventually come live as part of the compiler clap flags, but that's user-facing so I thought to err on the side of caution for now.